roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -60,10 +61,10 @@ bool ControlUtils::isPointProjectedOnLineSegment(const Vector2 &pointToCheck, co
                                                  const Vector2 &lineEnd) {
 
     Vector2 projectionPoint = pointToCheck.project(lineBegin, lineEnd);
-    double xMin = min(lineBegin.x, lineEnd.x);
-    double xMax = max(lineBegin.x, lineEnd.x);
-    double yMin = min(lineBegin.y, lineEnd.y);
-    double yMax = max(lineBegin.y, lineEnd.y);
+    double xMin = std::min(lineBegin.x, lineEnd.x);
```

Comment:
is there a reason you don't make this fmin and fmax like the rest of the changes you make?